docs/templates/resource_operation.md.j2

```diff
@@ -0,0 +1,44 @@
+{% if description %}
+## Description
+{{ description | escape_md_symbols }}
+{% endif %}
+
+## HTTP request
+```url
+{{ method | upper }} {{ url }}
+```
+{% if data_params %}
+
+## Data Parameters
```

Comment:
As templates for Data/Path/Query Parameters get duplicated in many places, we should start using the [Jinja's include](http://jinja.pocoo.org/docs/2.10/templates/#include) statement to follow DRY principle.

Reply:
done